it/uk/gov/hmrc/partnershipidentification/controllers/RegisterBusinessEntityControllerISpec.scala

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.partnershipidentification.controllers
+
+import play.api.http.Status.{BAD_REQUEST, INTERNAL_SERVER_ERROR, OK}
+import play.api.libs.json.Json
+import uk.gov.hmrc.partnershipidentification.assets.TestConstants._
+import uk.gov.hmrc.partnershipidentification.stubs.{AuthStub, RegisterWithMultipleIdentifiersStub}
+import uk.gov.hmrc.partnershipidentification.utils.ComponentSpecHelper
+
+
+class RegisterBusinessEntityControllerISpec extends ComponentSpecHelper with AuthStub with RegisterWithMultipleIdentifiersStub {
+
+  "POST /register" should {
+    "return OK with status Registered and the SafeId" when {
+      "the Registration was a success" in {
+        stubAuth(OK, successfulAuthResponse(Some(testInternalId)))
+        stubRegisterWithMultipleIdentifiersSuccess(testSautr)(OK, testSafeId)
+
+        val jsonBody = Json.obj(
+          "partnership" -> Json.obj(
+            "sautr" -> testSautr
+          )
+        )
+
+        val resultJson = Json.obj(
+          "registration" -> Json.obj(
+            "registrationStatus" -> "REGISTERED",
+            "registeredBusinessPartnerId" -> testSafeId))
+
+        val result = post("/register")(jsonBody)
+        result.status mustBe OK
+        result.json mustBe resultJson
+      }
+    }
+    "return INTERNAL_SERVER_ERROR with status Registration_Failed" when {
```

Comment:
This doesn't return an Internal Server Error anymore, maybe update to return OK with status Registration_Failed